web-ui/components/Menu.vue

```diff
@@ -22,6 +22,27 @@ import HeaderButton from "~/components/EditorPanelComponents/HeaderButton";
 
 export default {
   components: { HeaderButton },
+  mounted() {
+    // run run() on <C-Enter>
+    document.addEventListener("keydown", e => {
+      const modifierKey = navigator.platform.match("Mac")
+        ? e.metaKey
+        : e.ctrlKey;
+      if (e.key === "Enter" && modifierKey) {
```

Comment:
You can turn this into a method outside the `export default` and reuse it in both places:

```js
const isModifierPressed = event => navigator.platform.match("Mac") 
    ? e.metaKey
    : e.ctrlKey;

export default {
...
    if (e.key === "Enter" && isModifierPressed(e)) {
```